libs/libopen62541/Makefile

```
@@ -0,0 +1,103 @@
+#
+# Copyright (C) 2023 Michele Primavera
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libopen62541
+PKG_VERSION:=1.3.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/open62541/open62541.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=1587eba9e697591a342d91df02acb7460e74617895f810c006ad3d7e4bce21b8
+
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+CMAKE_INSTALL:=1
+
+define Package/libopen62541/config
+        source "$(SOURCE)/Config.in"
+endef
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+ifeq ($(CONFIG_HAS_MIPS16),y)
+  TARGET_CFLAGS += -fstack-check=generic
+endif
+
+define Package/libopen62541
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=An OPC UA library
+  URL:=https://open62541.org/
+  DEPENDS:=+librt +zlib +libstdcpp +libpthread +libatomic +LIBOPEN62541_MBDEDTLS:libmbedtls +LIBOPEN62541_OPENSSL:libopenssl
```

```suggestion
  DEPENDS:=+librt +zlib +libstdcpp +libpthread +libatomic \
    +LIBOPEN62541_MBDEDTLS:libmbedtls +LIBOPEN62541_OPENSSL:libopenssl
```

it's too long to write all dependencies to a single line